<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor</class>
 <widget class="QMainWindow" name="monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/image/bk2.jpg);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="register1">
     <attribute name="title">
      <string>Resistance</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);
border-image: transpoart;</string>
      </property>
      <property name="text">
       <string>Resistance</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>601</width>
        <height>381</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);
border-image: url(:/new/image/bk2.jpg);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_res">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);
border-image: transpoart;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_state">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);
border-image: transpoart;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);
border-image: transpoart;</string>
      </property>
      <property name="text">
       <string>State</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="res_update">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);
border-image: transpoart;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="camera">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
     <widget class="QLabel" name="image">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);
border-image: transpoart;</string>
      </property>
      <property name="text">
       <string>Recent Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_time">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>340</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(114, 159, 207);
border-image: transpoart;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cam_update">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>450</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;  border: 2px groove gray;

border-image: url(:/new/image/back.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
